deepcell_toolbox/processing.py

```diff
@@ -51,18 +51,21 @@ def normalize(image):
     return normal_image
 
 
-def phase_preprocess(image, kernel_size=64):
-    """Pre-process phase cytoplasm images using Contrast Limited Adaptive
+def histogram_norm_preprocess(image, kernel_size=64):
+    """Pre-process images using Contrast Limited Adaptive
     Histogram Equalization (CLAHE).
 
     Args:
         image (numpy.array): numpy array of phase image data.
         kernel_size (integer): Size of kernel for CLAHE.
 
     Returns:
-        numpy.array: Pre-processed phase image data with dtype float32.
+        numpy.array: Pre-processed image data with dtype float32.
     """
+    if not np.issubdtype(image.dtype, np.floating):
+        print('Converting image dtype to float')
```

Comment:
Maybe use a `warning` or a `logger` statement here instead of printing. I don't want to print to the console since this is a package that will be used in other locations.